frontend/src/__tests__/cypress/cypress/e2e/modelServing/ModelServingGlobal.cy.ts

```diff
@@ -572,4 +574,58 @@ describe('Model Serving Global', () => {
     modelServingGlobal.findDeployModelButton().click();
     cy.findByText('Error creating model server').should('not.exist');
   });
+
+  describe('Table filter', () => {
+    it('filter by name', () => {
+      initIntercepts({});
+      modelServingGlobal.visit('test-project');
+
+      // Verify initial run rows exist
+      modelServingGlobal.getModelRow('Test Inference Service').should('have.length', 1);
+
+      // Select the "Name" filter
+      TableToolbar.prototype.findFilterMenuOption('filter-dropdown-select', 'Name').click();
+      TableToolbar.prototype.findSearchInput().type('Test Inference Service');
+      // Verify only rows with the typed run name exist
+      modelServingGlobal.getModelRow('Test Inference Service').should('exist');
+      // Verify sort button works
+      modelServingGlobal.findSortButton('Model name').click();
+      modelServingGlobal.findSortButton('Model name').should(be.sortDescending);
+      modelServingGlobal.findSortButton('Model name').click();
+      modelServingGlobal.findSortButton('Model name').should(be.sortAscending);
+
+      // Search for non-existent run name
+      TableToolbar.prototype.findSearchInput().clear().type('Test Service');
+
+      // Verify no results were found
+      modelServingGlobal.findEmptyResults().should('exist');
+    });
+
+    it('filter by project', () => {
+      initIntercepts({
+        projectEnableModelMesh: true,
+      });
+      modelServingGlobal.visit('test-project');
+
+      // Verify initial run rows exist
+      modelServingGlobal.getModelRow('Test Inference Service').should('have.length', 1);
+
+      // Select the "Project" filter
+      TableToolbar.prototype.findFilterMenuOption('filter-dropdown-select', 'Project').click();
+      TableToolbar.prototype.findSearchInput().type('test project');
```

Comment:
same here.

Reply:
Looks like `TableToolbar` isn't correctly scoping its selectors. Please update at the same time.